modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Templates/index.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {useEffect} from 'react';
+import {useNavigate, useParams} from 'react-router-dom';
+
+import Container from '../../../../../components/Layout/Container';
+import ListViewRest from '../../../../../components/ListView';
+import {useHeader} from '../../../../../hooks';
+import i18n from '../../../../../i18n';
+import {filters} from '../../../../../schema/filter';
+import dayjs from '../../../../../util/date';
+import {searchUtil} from '../../../../../util/search';
+import useBuildTemplateActions from './useBuildTemplateActions';
+
+const BuildTemplates = () => {
+	const navigate = useNavigate();
+	const {projectId, routineId} = useParams();
+	const {actions} = useBuildTemplateActions();
+	const {setHeaderActions} = useHeader({
+		timeout: 110,
+		useTabs: [],
+	});
+	useEffect(() => {
+		setHeaderActions({actions});
```

Comment:
No need for this, since we want to put actions as empty array

You can set this in useHeader, the property name is: **useHeaderActions** as empty array